tests/acceptance/playwright.config.ts

```diff
@@ -1,132 +1,140 @@
-import { defineConfig, devices } from '@playwright/test';
-import path from 'path';
-import dotenv from 'dotenv';
+import path from "node:path";
+import { defineConfig, devices } from "@playwright/test";
+import dotenv from "dotenv";
 
 // Read from default ".env" file.
 dotenv.config();
 
-const missingEnvVars = ['APP_URL'].filter((envVar) => {
-    return process.env[envVar] === undefined;
+const missingEnvVars = ["APP_URL"].filter((envVar) => {
+	return process.env[envVar] === undefined;
 });
 
 if (missingEnvVars.length > 0) {
-    const envPath = path.resolve('.env');
+	const envPath = path.resolve(".env");
 
-    process.stdout.write(`Please provide the following env vars (loaded env: ${envPath}):\n`);
-    process.stdout.write('- ' + missingEnvVars.join('\n- ') + '\n');
+	process.stdout.write(
+		`Please provide the following env vars (loaded env: ${envPath}):\n`,
+	);
+	process.stdout.write(`- ${missingEnvVars.join("\n- ")}\n`);
 
-    process.exit(1);
+	process.exit(1);
 }
 
-process.env['SHOPWARE_ADMIN_USERNAME'] = process.env['SHOPWARE_ADMIN_USERNAME'] || 'admin';
-process.env['SHOPWARE_ADMIN_PASSWORD'] = process.env['SHOPWARE_ADMIN_PASSWORD'] || 'shopware';
+process.env.SHOPWARE_ADMIN_USERNAME =
+	process.env.SHOPWARE_ADMIN_USERNAME || "admin";
+process.env.SHOPWARE_ADMIN_PASSWORD =
+	process.env.SHOPWARE_ADMIN_PASSWORD || "shopware";
 
 const ignoreHTTPSErrors = process.env.SHOPWARE_PLAYWRIGHT_IGNORE_HTTPS_ERRORS === 'true' || process.env.SHOPWARE_PLAYWRIGHT_IGNORE_HTTPS_ERRORS === '1';
 
 if (process.env.DATABASE_URL) {
-    const matches = process.env.DATABASE_URL.match(/mysql:\/\/([^:@]+)(:([^:@]+))?@([^/]+)\/([^?]+)/);
-    if (matches) {
-        process.env.ATS_DATABASE_USERNAME = process.env.ATS_DATABASE_USERNAME || matches[1];
-        process.env.ATS_DATABASE_PASSWORD = process.env.ATS_DATABASE_PASSWORD || matches[3] || '';
-        process.env.ATS_DATABASE_HOST = process.env.ATS_DATABASE_HOST || matches[4];
-        process.env.ATS_DATABASE_NAME = process.env.ATS_DATABASE_NAME || matches[5];
-    }
+	const matches = process.env.DATABASE_URL.match(
+		/mysql:\/\/([^:@]+)(:([^:@]+))?@([^/]+)\/([^?]+)/,
+	);
+	if (matches) {
+		process.env.ATS_DATABASE_USERNAME =
+			process.env.ATS_DATABASE_USERNAME || matches[1];
+		process.env.ATS_DATABASE_PASSWORD =
+			process.env.ATS_DATABASE_PASSWORD || matches[3] || "";
+		process.env.ATS_DATABASE_HOST = process.env.ATS_DATABASE_HOST || matches[4];
+		process.env.ATS_DATABASE_NAME = process.env.ATS_DATABASE_NAME || matches[5];
+	}
 }
 
 // make sure APP_URL ends with a slash
-process.env['APP_URL'] = process.env['APP_URL'].replace(/\/+$/, '') + '/';
-if (process.env['ADMIN_URL']) {
-    process.env['ADMIN_URL'] = process.env['ADMIN_URL'].replace(/\/+$/, '') + '/';
+process.env.APP_URL = `${process.env.APP_URL.replace(/\/+$/, "")}/`;
```

Comment:
Why are you changing so much code style? Is this some of your personal IDE settings? I can't imagine that this is our general code style config. 🤔